### ISSUE TITLE
add ability to display text when no data is returned

### ISSUE THREAD

**Issue by hjalmers** [hjalmers/angular2-generic-table#24]

**Comment by rw-s** [comment]
It would be desirable if the "no data" text was a bindable property, so it can be customised for each use case (statically, or by the controller of the component that contains the table).

**Comment by rw-s** [comment]
Our internal issue ID: 196

**Comment by hjalmers** [comment]
@rw-s we've added the ability to show a custom message when table is loading data, when no data was returned or when filters/search returns no matches.

You can pass in you own texts using the `gtTexts` input like this:
 `<generic-table [gtTexts]="customTexts" ...></generic-table>`

where customTexts should be an object, currently this is the default:

```
{
    'loading':'Loading...',
    'no_data':'No data',
    'no_matching_data':'No data matching results found',
};
```
 
This will be available in our next release 1.0.0-rc.9
